assets/src/scss/layout/_navbar.scss

```diff
@@ -258,6 +258,10 @@ a.nav-link --nav-link-- {
   }
 }
 
+.btn-donate-top {
+  display: none;
```

Comment:
Why do we have to hide it? Can't we instead adjust the code to not add the class in case the feature is turned on? https://github.com/greenpeace/planet4-master-theme/blob/b61e9e0e039a77ad6db77596c2ca6f2a6194bbea/src/MasterSite.php#L553-L557

Reply:
Sure that works too :+1: